docs/topics/preparation.md

```diff
@@ -77,19 +77,21 @@ export HCLOUD_WORKER_MACHINE_TYPE=cpx31
 * HCLOUD_IMAGE_NAME: The Image name of your operating system.
 * HCLOUD_X_MACHINE_TYPE: https://www.hetzner.com/cloud#pricing
 
-For a list of all variables needed for generating a cluster manifest (from the cluster-template.yaml), use `clusterctl generate cluster my-cluster --list-variables`:
+For a list of all variables needed for generating a cluster manifest (from the cluster-template.yaml), use `clusterctl generate cluster --infrastructure hetzner:v1.0.0-beta.33 --list-variables hetzner-cluster`:
```

Comment:
@kranurag7 do we have some release procedures for CAPH? You need to add that we have to manually go through the docs and see where we have to bump the release

Reply:
I think instead of bumping version all the time, we can add a placeholder something like `<caph-version>` and then the user will get the latest version by checking the latest release via GitHub.

Reply:
@guettli what is your opinion here? I have none!

Reply:
I think using `vX.X.X` here is fine. Some lines above is a hyperlink to the latest version.